sites/automationworld/redirect-handler.js

```diff
@@ -0,0 +1,15 @@
+const redirect = (from, pattern) => pattern.exec(from);
+
+module.exports = ({ from }) => {
+  const digital = redirect(from, /\/sites\/default\/files\/digital_edition\/(.*)/);
+  if (digital && digital[1]) {
+    return { to: `https://digitaleditions.automationworld.com/${digital[1]}` };
+  }
+
+  const s3 = redirect(from, /\/sites\/default\/files\/(.*)/);
+  if (s3 && s3[1]) {
+    return { to: `https://s3.us-east-2.amazonaws.com/pmg-production/Migrated+-+DO+NOT+USE/AW/${s3[1]}` };
```

Comment:
I'm confused by this redirect... can you explain why it's going to a bucket with such a "scary" pathname? i.e. `Migrated+-+DO+NOT+USE`

Reply:
This what not our call, that is PMMI's bucket. Guessing they are old assets they don't want people to put stuff in there anymore.